src/components/Visits/index.js

```diff
@@ -74,18 +74,14 @@ const Visits = () => {
     if (sortKey && sortDir) {
       visits.sort((a, b) => {
         let answer;
-        if (sortKey !== "doctor") {
-          if (sortDir === "ASC") {
-            answer = a[sortKey] < b[sortKey] ? 1 : -1;
-          } else if (sortDir === "DESC") {
-            answer = a[sortKey] > b[sortKey] ? 1 : -1;
-          }
-        } else if (sortKey === "doctor") {
-          if (sortDir === "ASC") {
+        if (sortDir === "ASC") {
+          if (sortKey === "doctor") {
```

Comment:
the duplication of the code has not gone away and I do not understand why there is a check for only one key?